Test case test_connection_time_out failed in MS provider

The test case given below is failing in Managed Services provider cluster.
tests/managed-service/test_post_installation_state.py::TestPostInstallationState::test_connection_time_out

This is because the bug 2073025 was recently closed as a duplicate of 2117312. The bugzilla marker should be updated to skip the test.
```
self = <test_post_installation_state.TestPostInstallationState object at 0x7fc4abf79310>

    @tier1
    @bugzilla("2073025")
    @runs_on_provider
    @pytest.mark.polarion_id("OCS-2695")
    @managed_service_required
    def test_connection_time_out(self):
        """
        Test that connection from mon pod to external domain is blocked and gets timeout
        """
        mon_pod = pod.get_mon_pods()[0]
        with pytest.raises(CommandFailed) as cmdfailed:
>           mon_pod.exec_cmd_on_pod("curl google.com")
E           Failed: DID NOT RAISE <class 'ocs_ci.ocs.exceptions.CommandFailed'>

tests/managed-service/test_post_installation_state.py:170: Failed
```

--- a/tests/managed-service/test_post_installation_state.py
+++ b/tests/managed-service/test_post_installation_state.py
@@ -157,7 +157,7 @@ class TestPostInstallationState(ManageTest):
         assert len(log_lines) > 100
 
     @tier1
-    @bugzilla("2073025")
+    @bugzilla("2117312")
     @runs_on_provider
     @pytest.mark.polarion_id("OCS-2695")
     @managed_service_required

